lesson5/src/Zadanie25/Bankomat.java

```diff
@@ -0,0 +1,150 @@
+//Создать класс и объекты, описывающие Банкомат
+//
+//        Набор купюр, находящихся в банкомате должен задаваться 3мя свойствами: +
+//        * количеством купюр номиналом 20
+//        * -#-#-#-#                    50
+//        * -#-#-#-#                    100
+//
+//        Сделать методы для добавления денег в банкомат ++
+//
+//        Сделать, функцию снимающую деньги:
+//        На вход передается сумма денег, на выход - булевское значение(операция удалась или нет) ++
+//        При снятии денег, функция должна распечатывать, каким количством купюр, какого номинала выдается сумма ++
+//
+//        Создать конструктор с тремя параметрами - количеством купюр++
+
+package Zadanie25;
+
+public class Bankomat {
+
+    int amount_20;
+    int amount_50;
+    int amount_100;
+    int summa;
+
+    public int summa(){
+        summa = ((amount_20*20) + (amount_50*50) + (amount_100*100));
+        return summa;
+    }
+
+    public void inset_money(int inset_20, int inset_50, int inset_100){
+        amount_20 +=inset_20;
+        amount_50 +=inset_50;
+        amount_100 +=inset_100;
+
+    }
+
+    public void inset_money(int inset_summa){
+        if((inset_summa <40 && inset_summa!=20) || inset_summa%10 !=0){System.out.println("Такие купюры не принимаю");}
+        else {
+            if(inset_summa - (inset_summa/100)*100 == 10){
+                amount_100 += (inset_summa/100 -1) ;
+                amount_50++;
+                amount_20 +=3;
+            }
+            else {
+                if(inset_summa - (inset_summa/100)*100 == 30){
+                    amount_100 =(inset_summa/100 -1) ;
+                    amount_50++ ;
+                    amount_20 +=4 ;
+                }
+                else{
+            amount_100 += inset_summa / 100;
+            amount_50 += (inset_summa - (inset_summa / 100)*100) / 50;
+            amount_20 += (inset_summa - ((inset_summa / 100)*100) - ((inset_summa - (inset_summa / 100)*100) / 50) * 50) / 20;
+        }}}
+    }
+
+    public boolean get_money(int get_summa){
+        boolean operation = false;
+        if(get_summa<=summa() && (get_summa>30||get_summa==20) && get_summa%10==0){
+            operation=true;
```

Comment:
set true for operation result before operation start? I don't think it's good idea